style.css

```diff
@@ -0,0 +1,717 @@
+    padding: 8px 10px;
+}
+.blog__info {
+    display: flex;
+    margin-bottom: 10px;
+    font-size: 14px;
+    line-height: 20px;
+    font-weight: 600;
+}
+.blog__autor {
+    margin-right: 20px;
+}
+.blog__autor::after {
+    content: "";
+    position: absolute;
+    top: 2px;
+    right: -14px;
+    z-index: 2;
+    width: 2px;
+    height: 16px;
+    background-color: rgba(208, 215, 251, 0.17);
+}
+.blog__autor,
+.blog__info time {
+    position: relative;
+    padding-left: 26px;
+}
+.blog__autor::before,
+.blog__info time::before {
+    content: "";
+    position: absolute;
+    top: 2px;
+    left: 4px;
+    z-index: 2;
+    width: 16px;
+    height: 16px;
+    background-image: url(img/user.svg);
+}
+.blog__info time::before {
+    background-image: url(img/calendar.svg);
+}
+.blog__name {
+    line-height: 48px;
+    font-size: 36px;
+    margin-bottom: 6px;
+    color: #F5F8FB;
+}
+.blog__item p {
+    margin-bottom: 16px;
+}
+.blog__button {
+    display: flex;
+}
+.align-center {
+    text-align: center;
+}
+.align-center .btn{
+    display: inline-block;
+}
+.about {
+    max-width: 960px;
+    margin: 0 auto 60px;
+    color: rgba(239, 245, 250, 0.82);
+}
+.about__subtitle {
+    font-weight: 500;
+    font-size: 20px;
+    text-align: center;
+    margin-bottom: 40px;
+}
+.about__list {
+    list-style: none;
+    counter-reset: my-awesome-counter;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 0 40px;
+    padding: 0;
+    gap: 20px;
+}
+.about__item {
+    counter-increment: my-awesome-counter;
+    background-color: #1A1925;
+    flex-basis: calc(50% - 20px);
+    position: relative;
+}
+.about__link {
+    display: block;
+    text-decoration: none;
+    color: #F5F8FB;
+    padding: 80px 20px 20px;
+    position: relative;
+    z-index: 2;
+}
+.about__link::after,
+.about__link::before {
+    content: "";
+    position: absolute;
+    display: none;
+    top: 0;
+    left: 0;
+    z-index: 2;
+    width: 70px;
+    height: 90px;
+    background-image: url(img/about-shadow-top.png);
+}
+.about__link::before {
+    top: auto;
+    left: auto;
+    bottom: 0;
+    right: 0;
+    width: 80px;
+    height: 104px;
+    background-image: url(img/about-shadow-botton.png);
+}
+.about__link:hover::after,
+.about__link:hover::before {
+    display: block;
+}
+.about__item::before {
+    content: "0" counter(my-awesome-counter);
+    position: absolute;
+    top: 24px;
+    left: 20px;
+    z-index: 1;
+    font-size: 80px;
+    font-family: "Bakbak One", Times New Roman, Georgia, serif;
+    line-height: 96px;
+    color: #1A1925;
+    -webkit-text-stroke: 1px #0265DC;
+}
+.about__item p {
+    margin-bottom: 20px;
+}
+.team{
+    max-width: 1300px;
+    margin: 0 auto 40px;
+    padding: 0 15px;
+    color: rgba(239, 245, 250, 0.82);
+}
+.team__list {
+    gap: 30px; 
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center; 
+}
+.team__item {
+    background-color: #1D1C28;
+    flex-basis: 293px;
```

Comment:
How I said in another comment, this is design bug and width must be 294px.
As you can see on the layout and screenshots, our icons do not fit exactly 1px into their container
<img width="651" alt="image" src="https://github.com/user-attachments/assets/ee9ca583-7826-4021-8fa0-026ec0e57bb6">
<img width="691" alt="image" src="https://github.com/user-attachments/assets/5076023d-f193-456c-85a2-06015bfc024a">